src/index.html

```diff
@@ -17,6 +17,8 @@
       integrity="sha512-ISS7cAi1PEhQ8jnbJpJZMd29NlhNj4AWYyLOSp2CE/CsHxTCu+r+t0D2yoJudVrd0/8fTVPUVDzY5Tvli75u/g=="
       crossorigin="anonymous"
     />
+
+    <link rel="stylesheet" href="./src/sass/main.css" />
```

Comment:
@AnastazjaNikolaBokun You shouldn't include styles in HTML file, styles are automatically imported in index.js.

Remove this line